dev_tools/lib/solidus/release_drafter.rb

```diff
@@ -68,7 +68,6 @@ def save_release(draft, candidate_tag, branch)
     def full_changelog(current_diff_source_tag, candidate_tag)
       <<~TXT
 
-
```

Comment:
This extra space is intentional, see https://app.circleci.com/pipelines/github/solidusio/solidus/3952/workflows/a82c4d50-c4ee-426b-92b4-5cccee0ad4fe/jobs/36006. We can probably get rid of it, but the specs should reflect the change, cc @waiting-for-dev.

Reply:
Ha, good catch.